recreate-cert-for-api.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -x
+
+do_clean(){
+    rm -fr "$workdir"
+}
+
+trap do_clean EXIT
+
+cert_subject=dummy.recert.name
```

Comment:
Pass the actual domain to the script and use it here

Reply:
Don't :smile: